releng/config-forker/main.go

```diff
@@ -351,6 +353,23 @@ func generateNameVariant(name, version string, generic bool) string {
 	return strings.ReplaceAll(name, "-master", suffix)
 }
 
+func generatePresubmitNameVariant(name, version string) string {
+	suffix := "-" + version
+	if !strings.HasSuffix(name, "-master") {
+		return name + suffix
```

Comment:
Note that I follow in this line the example of `generateNameVariant` which works exactly like this.
I would have expected that we would append the suffix and then still do a `ReplaceAll`:

```go
func generatePresubmitNameVariant(name, version string) string {
	suffix := "-" + version
	if !strings.HasSuffix(name, "-master") {
		name = name + suffix
	}
	return strings.ReplaceAll(name, "-master", suffix)
}
```